examples/server/ktor-server/build.gradle.kts

```diff
@@ -14,6 +14,7 @@ application {
 
 val kotlinCoroutinesVersion: String by project
 dependencies {
+    implementation("com.expediagroup", "graphql-kotlin-server")
```

Comment:
lets follow consistent pattern -> either single string or change to explicit parts but lets not mix it